canonicalwebteam/blog/django/urls.py

```diff
@@ -50,6 +51,7 @@ def to_url(self, value):
     path(r"/<yyyy:year>/<mm:month>/<str:slug>", article_redirect),
     path(r"/<yyyy:year>/<str:slug>", article_redirect),
     path(r"/archives", archives),
+    path(r"/author/<str:username>", author),
```

Comment:
I'm wondering... should this be "/authors"? 'Cos I think we have "/topics" right? Plurals in general is the recommendation in https://pages.apigee.com/rs/apigee/images/api-design-ebook-2012-03.pdf and I think makes sense in more use-cases than singular.

Do you agree?